packages/xo-web/src/xo-app/settings/remotes/remote.js

```diff
@@ -476,6 +501,83 @@ export default decorate([
               </div>
             </fieldset>
           )}
+          {(type === 'azure' || type === 'azurite') && (
+            <fieldset className='form-group form-group'>
+              {state.type === 'azurite' && (
+                <div className='input-group form-group'>
+                  <span className='align-middle '>
+                    {_('remoteS3LabelAllowInsecure')}{' '}
```

Comment:
```suggestion
                    {_('remoteAzureLabelAllowInsecure')}{' '}
```

Reply:
Maybe we could change s3 and azure to remote to avoid repeating translations as they are the same?